crates/next-api/src/nft_json.rs

```diff
@@ -123,6 +208,44 @@ impl Asset for NftJsonAsset {
             .copied()
             .chain(std::iter::once(chunk))
             .collect();
+
+        let exclude_globs: Option<Vec<Glob>> = if let Some(route) = &this.page_name {
+            let project_path = this.project.project_path().await?;
+
+            if let Some(excludes_config) = output_file_tracing_excludes {
+                let mut combined_excludes: BTreeSet<RcStr> = BTreeSet::new();
+
+                if let Some(excludes_obj) = excludes_config.as_object() {
+                    for (glob_pattern, exclude_patterns) in excludes_obj {
+                        // Check if the route matches the glob pattern
+                        if let Ok(glob) = Glob::parse(glob_pattern)
```

Comment:
glob parsing is non trivial, so useing the `Glob::new(...).await` API might be more efficient since we can cache across executions

also, i assume we should be reporting issues for invalid globs instead of ignoring them.  should we simply abort if the glob doesn't parse?

Reply:
Yeah it should error when the glob is not valid.